baseapp/abci.go

```diff
@@ -540,6 +540,15 @@ func gRPCErrorToSDKError(err error) error {
 // createQueryContext creates a new sdk.Context for a query, taking as args
 // the block height and whether the query needs a proof or not.
 func (app *BaseApp) createQueryContext(height int64, prove bool) (sdk.Context, error) {
```

Comment:
nit, can you update the comment from above to state that height = 0 injects the latest height to the context?

Reply:
I am not familiar with this code though, and will spend time with a back and forth for the right wording.